package.json

```diff
@@ -8,10 +8,30 @@
     "node": ">=6.13.0"
   },
   "scripts": {
```

Comment:
I would recommend alphabetizing key-value pairs in an object unless you have a compelling reason to do otherwise.